vendor/github.com/vishvananda/netlink/Makefile

```diff
@@ -3,7 +3,8 @@ DIRS := \
 	nl
 
 DEPS = \
-	github.com/vishvananda/netns
```

Comment:
Is it possible to separate these vendor into separate PR? Its bit harder to do review along with these changes.

Reply:
@SaranBalaji90 It won't compile if I put them into separate PRs.  Right now, there are in 2 different commits, one for cni and one for vendor update